src/pages/AccountPage/ProfilePage.jsx

```diff
@@ -124,15 +127,46 @@ const ProfilePage = ({ user = {}, isLoading }) => {
     }
   }
 
+
+  const onUpdateAvatar = async (file) => {
```

Comment:
I would move this into the Avatar component, then that component can be more easily reused, along with it's updating logic. No reason to live inside ProfilePage?

Reply:
Done.